### PR TITLE
AWS CloudWatch Exporter docs update

### PR DESCRIPTION
**Pull request by aDaemonThread**

### CODE REVIEW

**Review by vordimous — DISMISSED** [review]

File: src/reference/config/telemetry/exporters/exporter-aws.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+Zilla runtime aws exporter.
+
+```yaml {3}
+exporters:
+  aws0:
+    type: aws
+    options:
+      metrics:
+        namespace: zilla
```
Comment (jfallows) [inline src/reference/config/telemetry/exporters/exporter-aws.md:23]:
```suggestion
        namespace: zilla-metrics
```

File: src/reference/config/telemetry/exporters/exporter-aws.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+### options
+
+> `object`
+
+`aws`-specific options.
+
+```yaml
+options:
+  metrics:
+    namespace: zilla
```
Comment (jfallows) [inline src/reference/config/telemetry/exporters/exporter-aws.md:61]:
```suggestion
    namespace: zilla-metrics
```

**Review by jfallows — CHANGES_REQUESTED** [review]

**Review by jfallows — APPROVED** [review]

**Review by jfallows — COMMENTED** [review]
@aDaemonThread please rename `aws` exporter to `aws-cloudwatch` exporter.

**Review by jfallows — COMMENTED** [review]
@vordimous please hold off on merging until this has been renamed.

File: src/reference/config/telemetry/exporters/exporter-aws.md
Comment (jfallows) [inline src/reference/config/telemetry/exporters/exporter-aws.md:1]: Please rename file to `exporter-aws-cloudwatch.md`.

**Review by jfallows — CHANGES_REQUESTED** [review]

**Review by vordimous — CHANGES_REQUESTED** [review]

**Review by jfallows — APPROVED** [review]

**Pull request merged**